test/test_lazy.py

```diff
@@ -0,0 +1,90 @@
+import torch
+from common_utils import TestCase, run_tests, skipIfRocm, do_test_dtypes, do_test_empty_full, load_tests
+import time
+
+class TestLazy(TestCase):
+  def test_bench(self):
+    a = torch.rand(1,1)
+    b = torch.rand(1,1)
+    a_ = a.to_lazy()
+    b_ = b.to_lazy()
+    iters = int(1e2)
+
+    t = time.time()
+    for _ in range(iters):
+      c = a + b
+      a = c
+    print(time.time() - t)
```

Comment:
Probably don't want print statements in the test suite (I know this is intended to be a benchmarky thing, but for tests people want them to run as fast as possible.) Also... 10 iters is really short :>